src/dosext/mfs/mfs.c

```diff
@@ -2616,10 +2668,17 @@ int ResetRedirection(int dsk)
   return 0;
 }
 
+/* See if there is a physical drive behind any redirection */
+static int hasPhysical(cds_t cds)
+{
+  far_t DPBptr = cds_DPB_pointer(cds);
+
+  return (DPBptr.offset != 0 || DPBptr.segment != 0);
```

Comment:
Is this info unavailable from cds flags?

Reply:
Nope, flags were overwritten when the drive was redirected. I mostly just moved this existing test into a function as I need to use it twice.

Reply:
Isn't it a bug?
`cds_flags(cds) = CDS_FLAG_READY | CDS_FLAG_REMOTE | CDS_FLAG_NOTNET;`

Why not `cds_flags(cds) |= CDS_FLAG_READY | CDS_FLAG_REMOTE;`
instead?

Reply:
Could be and seems a strange combination if READY really equals PHYSICAL